vignettes/analysing-n-grams.Rmd

```diff
@@ -125,11 +125,12 @@ research_articles <- imported_metadata %>%
   filter(!str_detect(article_title, "^Book Review"))
 ```
 
-
+## Filtering articles
+E.g.: It can happen that for one specific filter, one has to search in more than one column (captain obvious?)
 
 ## The moving wall - filtering articles by time
 Since JSTOR has a [moving wall](https://support.jstor.org/hc/en-us/articles/115004879547-JSTOR-s-Moving-Wall-Archive-vs-Current-Definitions),
-we could take a look at the number of articles per year in our dataset.
+we should take a look at the number of articles per year in our dataset. (#3)
```

Comment:
You can do so even without the moving wall of JSTOR, I think - separate the information of the moving wall from the description of the intention behind what you are doing - therefore "we should". You could also start from the other end, doing sth like: "let's just have a look at the number of articles per year - oh, something quirky in recent years, why so? - oh, a moving wall, let's cut the recent years"
in the end one could also look at the last year in which articles appeared in every journal in the dataset, as long as it is safe to assume there is at least one article published per year and journal, possibly deliberately excluding journals who....went bankrupt or sth ;)